TestingApp/ViewController/ViewController.swift

```diff
@@ -403,7 +406,10 @@ extension ViewController {
 
     @IBAction private func toggleAuthentication() {
         catchingError {
-            try Glia.sharedInstance.configure(with: configuration) { [weak self] in
+            try Glia.sharedInstance.configure(
```

Comment:
`configure` with `queueId` is deprecated

Reply:
Fixed, i added the result also a17158aa2793fe617819543052d53269ad7c8fd6